src/scipp/serialization.py

```diff
@@ -6,7 +6,7 @@
 
 
 def serialize(var: Union[Variable, DataArray, Dataset]) -> Tuple[Dict, List[bytes]]:
-    """Serialize scipp object."""
+    """Serialize Scipp object."""
```

Comment:
Having seen more cases, I feel we should not change this in the py files, but:

- When we mean the *project*, we write Scipp.
- When we mean the *package* or *module* we write scipp?

Would this makes sense @jl-wynen @nvaytet?

Reply:
Where do we draw the line between the two? In this case, 'Scipp' cna just as well refer to the project. I don't see a reason to distinguish the two unless we refer specifically to code. In that case, we can use `scipp`.

Reply:
Personally I really don't mind/care that much. Maybe it should just be either Scipp or `scipp` (i.e. don't use scipp lowercase without the code tags around it).
Then, where one should be used and not the other is up to you, I don't mind.